models/bamf_ct_liver_tumor/config/default.yml

```diff
@@ -0,0 +1,53 @@
+general:
+  data_base_dir: /app/data
+  version: 1.0
+  description: default configuration for Bamf CT Liver and tumor segmentation (dicom to dicom)
+
+execute:
+- DicomImporter
+- NiftiConverter
+- NNUnetRunnerV2
+- PostProcessor
+- DsegConverter
+- DataOrganizer
+
+modules:
+  DicomImporter:
+    source_dir: input_data
+    import_dir: sorted_data
+    sort_data: true
+    meta: 
+      mod: '%Modality'
+
+  NiftiConverter:
+    engine: dcm2niix
+
+  NNUnetRunnerV2:
+    in_data: nifti:mod=ct
+    nnunet_dataset: Dataset006_Liver
+    nnunet_config: 3d_fullres
+    roi: SPLEEN,KIDNEY,GALLBLADDER,DUODENUM,PANCREAS,SMALL_INTESTINE,LUNG,LIVER,LIVER+NEOPLASM_MALIGNANT
+    export_prob_maps: False
+
+  DsegConverter:
+    model_name: bamf_ct_liver_tumor
+    target_dicom: dicom:mod=ct
+    source_segs: nifti:mod=seg:processor=bamf
+    skip_empty_slices: True
+
+  DataOrganizer:
+    targets:
+    - dicomseg-->[i:sid]/bamf_ct_liver_tumor.seg.dcm
+
+segdb:
```

Comment:
Are these annotations project specific? If not, I'd add them to SegDB too?

Reply:
@LennyN95 segdb has `LIVER+NEOPLASM_MALIGNANT_PRIMARY`, but we want it to specify primary or metestastasis, just `LIVER+NEOPLASM_MALIGNANT` would be ideal. That could link to a code like [C190593](https://thesaurus.cancer.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&version=24.07e&ns=ncit&code=C190593&key=n1993599127&b=1&n=null). what do you think?

Reply:
Updated label same as [comment](https://github.com/MHubAI/models/pull/84#discussion_r1722420985)